lib/elixir/test/elixir/option_parser_test.exs

```diff
@@ -256,6 +256,12 @@ defmodule OptionParserTest do
            == {[foo: true, boo: "-"], ["-"], []}
   end
 
+  test "allow nonexistent atom`s" do
+    opts = [switches: [a: :string], allow_nonexistent_atoms: true]
+    assert OptionParser.parse(["--a", "b", "--c", "d", "--option-key-creates-atom", "f"], opts)
```

Comment:
Ah, didn't notice it's in the whole file. Let's stay consistent with the rest of the file then and we can think about uniforming later.